README.md

```diff
@@ -39,7 +39,7 @@ building an image:
 
    Or build with an Accumulo tarball (located in same directory as DockerFile) using the command below:
 
-        docker build --build-arg ACCUMULO_FILE=accumulo-2.1.0-SNAPSHOT-bin.tar.gz -t accumulo .
+        docker build --build-arg ACCUMULO_FILE=accumulo-2.1.2-SNAPSHOT-bin.tar.gz -t accumulo .
```

Comment:
Since this is just an example, it doesn't need to be updated every time.

```suggestion
        docker build --build-arg ACCUMULO_FILE=accumulo-*-bin.tar.gz -t accumulo .
```

Reply:
what about replacing `*` with `<version>` in your suggestion?

Reply:
That's fine, or something like `/path/to/accumulo-<version>-bin.tar.gz`
The main thing is that it should either be something that somebody can copy/paste directly (with `*`, that should work), or it should be very obvious what parts they need to edit (which `<version>` would provide).

Reply:
Addressed in 9930f8e. Also, unrelated, I added `json` to a code block in the readme for better markdown styling.